server/utils/auth/google_client.py

```diff
@@ -119,5 +119,16 @@ def create_event_type(self, duration, title, start_time, timezone, host_email, b
         }
 
         event_service = build('calendar', 'v3', credentials=self.credentials)
-        event = event_service.events().insert(calendarId='primary', body=event).execute()
-        # print('Event created: %s' % (event.get('htmlLink')))
\ No newline at end of file
+        event = event_service.events().insert(calendarId='primary', body=event, sendUpdates='all').execute()
+        if event.get('id') is None:
+            return None
+        return event['id']
+        # print('Event created: %s' % (event.get('htmlLink')))
+    
+    def delete_calendar_event(self,google_event_id):
+        try:
+            service = build('calendar', 'v3', credentials=self.credentials)
+            service.events().delete(calendarId='primary', eventId=google_event_id).execute()
```

Comment:
@fangyu3 I changed the `eventId` here because I noticed it was `eventId='event_id'` and I think it should be `eventId=google_event_id`. Please confirm and I'll merge then ✋

Reply:
yep good catch!